### ISSUE TITLE
Bots disabling not working

### ISSUE THREAD

**Issue by boncea** [WhichBrowser/Parser-PHP#97]
Hello,

I tried recently to disable the bot detection, as I want the bots to be redirected according to the device type. I found out that this is not working as it's supposed to do from the documentation. Then I checked the inner code and found out that the tests might be wrong as well.

For example in `whichbrowser/parser/tests/data/bots/disabled.yaml` we have
```
    detectBots: false
    headers: 'User-Agent: Mozilla/5.0 (iPhone; CPU iPhone OS 6_0 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/6.0 Mobile/10A5376e Safari/8536.25 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'
    result: { browser: { name: Googlebot, version: '2.1' }, device: { type: bot } }
    readable: 'Googlebot 2.1'
```

As per my understanding, the expected device type here should be `mobile`, not `bot`, because the bot detection is supposed to be disabled.
The check is done in `whichbrowser/parser/src/Analyser/Header/Useragent.php:29` where it's looking for an `options` property, but I couldn't find anywhere the `UserAgent` class to receive that `options`, so I think it should just be passed from `whichbrowser/parser/src/Analyser/Header.php:85`, in the same way as the `data`.

I'll come up with a pull request, so I'm waiting for a review. I hope I didn't misunderstood something. Please let me know about your thoughts. 

Thank you!

**Comment by NielsLeenheer** [comment]
You were absolutely correct. The options needed to be passed on, but wasn't. Merged you PR as is and will release an update soon. Thanks again!

**Comment by mariotsi** [comment]
Hi @boncea thanks for pointing this out. I found this issue also on Parser-Javascript,
The root cause was different but with the same effect. https://github.com/WhichBrowser/Parser-JavaScript/pull/26

It will be fixed with the version 0.3.3

**Comment by boncea** [comment]
Hi @NielsLeenheer, thank you for merging the PR. I'm waiting for the new release! @mariotsi you're welcome, maybe it's a feature that people don't use a lot.
